src/test/suite/fixtures/recorded/languages/typescript/clearState.yml

```diff
@@ -0,0 +1,31 @@
+languageId: typescript
+command:
+  spokenForm: clear state
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: statement}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |-
+    interface Foo {
+      bar: Bar;
+      baz: Baz;
+    }
+  selections:
+    - anchor: {line: 1, character: 2}
+      active: {line: 1, character: 2}
+  marks: {}
+finalState:
+  documentContents: |-
+    interface Foo {
+      ;
```

Comment:
Could go either way on keeping this semicolon for "change", but it should definitely go for "chuck"

Reply:
That makes it a bit more complicated than just treating it like any other statement but I do agree

Reply:
done